packages/manager/modules/pci/src/projects/new/new.html

```diff
@@ -1,7 +1,7 @@
 <div class="pci-projects-new h-100">
-    <div class="container-fluid h-100">
-        <div class="row h-100 justify-content-md-center">
-            <div class="col-md-7 p-5 bg-white h-100">
+    <div class="container-fluid overflow-auto h-100">
+        <div class="row minh-100 justify-content-md-center">
```

Comment:
If you do that, i think the navbar isn't sticky anymore, right ?

Reply:
Nope, it's okay with that ;-)

Reply:
I think we don't need that because develop has been fixed in another way :)

Reply:
Nope, the white background is not extended if there is an overflow.
I have merged this branch into an other one, to keep everything at one plae. I close this PR.